doc/sphinx-guides/source/qa/overview.md

```diff
@@ -0,0 +1,46 @@
+# Overview
+
+```{contents} Contents:
+:local: 
+:depth: 3
+```
+
+## Introduction
+
+This guide describes the testing process used by QA at IQSS and provides a reference for others filling in for that role. Please note that many variations are possible, and the main thing is to catch bugs and provide a good quality product to the user community.
+
+## Workflow
+
+The basic workflow is as follows. Bugs or feature requests are submitted to GitHub by the community or by team members as [issues](https://github.com/IQSS/dataverse/issues). These issues are prioritized and added to a two-week sprint that is reflected on the GitHub {ref}`kanban-board`. As developers work on these issues, a GitHub branch is produced, code is contributed, and a pull request is made to merge these new changes back into the common {ref}`develop branch <develop-branch>` and ultimately released as part of the product.
+
+Before a pull request is moved to QA, it must be reviewed by a member of the development team from a coding perspective, and it must pass automated tests. There it is tested manually, exercising the UI (using three common browsers) and any business logic it implements.  
```

Comment:
"Once a pull request has been submitted the code and any additional documentation or accompanying release notes are reviewed by another developer. During this code review the reviewer may suggest coding or documentation  changes to the original developer. When the reviewer and original developer are satisfied with the code and the PR passes automated tests, the reviewer with approve the pull request and mark it as "Ready for QA". In QA the PR is tested manually via the UI  (using three common browsers) to verify business logic it implements."